crates/libgraphql-core/src/test/snapshot_tests/mod.rs

```diff
@@ -0,0 +1,64 @@
+//! Snapshot testing framework for validating GraphQL schemas and operations.
+//!
+//! This module provides a file-based testing system that validates libgraphql
+//! against known-good and known-invalid GraphQL schemas and operations.
+//!
+//! For detailed usage instructions, see the [README](snapshot_tests/fixtures/README.md).
+
+mod snapshot_test_case;
+mod test_runner;
+
+#[cfg(test)]
+mod tests {
+    use super::test_runner;
+    use std::path::Path;
+    use std::path::PathBuf;
+    use std::sync::OnceLock;
+
+    fn get_fixtures_dir() -> &'static Path {
+        static FIXTURES_DIR: OnceLock<PathBuf> = OnceLock::new();
+        FIXTURES_DIR.get_or_init(|| {
+            PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("src/test/snapshot_tests/fixtures")
+        })
+    }
+
+    #[test]
+    fn verify_snapshot_test_schemas() {
```

Comment:
Add a comment that this `#[test]` finds and runs all snapshot tests and is thus meant to be the only one (no need to add others for, say, a newly added test/test suite)